README.md

```diff
@@ -218,7 +217,7 @@ export INPUT_GITHUB_TOKEN=$(printenv GITHUB_TOKEN)
 export INPUT_LIV_DOC_REGIME=true
 export INPUT_VERBOSE_LOGGING=true
 
-# Environment variables for Living Documentation regime functionality
+# Environment variables for LivDoc regime functionality
```

Comment:
```suggestion
# Environment variables for Living Documentation (LivDoc) regime functionality
```

Reply:
Part of next issue: https://github.com/AbsaOSS/living-doc-generator/issues/75#issue-2735478783.